client/src/components/admin/AdminManagePlans.jsx

```diff
@@ -242,7 +239,7 @@ const AdminManagePlans = () => {
                   >
                     <td className="text-center">{index + 1}</td>
                     <td className="text-center">{plan.planName}</td>
-                    <td className="text-center">₹ {plan.amount}</td>
+                    <td className="text-center">{plan.amount}</td>
```

Comment:
May I know why did you remove this?

Reply:
will do it. thanks!

Reply:
Why did you merge without resolving them?